### PR TITLE
Adding ACON capture and setting up for server upgrade

### PR DESCRIPTION
**Pull request by lisazacarias**
Preliminary simulacrum testing successful

### CODE REVIEW

File: gui_utils.py
Comment (nneveu) [inline gui_utils.py:1]: Do other files use this function? Is that why it was pulled out?

File: setup_linac.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+        self.status_message = f"Turning {self} off"
+        self.turnOff()
+
+        self.status_message = f"Turning on {self} SSA if not on already"
+        self.ssa.turn_on()
+
+        self.status_message = f"Resetting {self} interlocks"
+        self.reset_interlocks()
+
```
Comment (nneveu) [inline setup_linac.py:121]: Is this section a candidate for case/match instead?

**Review by nneveu — APPROVED** [review]

File: gui_utils.py
Comment (lisazacarias) [reply gui_utils.py:1]: Both setup_gui and setup_linac use it so I figured it made sense. Also just wanted to remove bloat from the GUI file

File: setup_linac.py (hunk 1 above)
Comment (lisazacarias) [reply setup_linac.py:121]: All the ifs? Unless I'm misinterpreting your point, they're not exclusive. Like, sometimes you want all of them to run, sometimes you want a subset to run, so I think figuring out all the permutations would make for _a lot_ of cases right?

Comment (nneveu) [reply setup_linac.py:121]: Yep I meant the ifs. Ok, makes sense if they aren't exclusive.

**Review by s-aderhold — APPROVED** [review]
looks good

**Pull request merged**